submarine-server/server-core/pom.xml

```diff
@@ -360,6 +360,31 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- Jason Web Token -->
+    <dependency>
+      <groupId>io.jsonwebtoken</groupId>
+      <artifactId>jjwt-api</artifactId>
+      <version>0.11.2</version>
```

Comment:
All the dependencies should be moved to project pom.xml and use the defined version property.